File: src/main/java/com/example/javakotlindemo/app/AppMain.java
package com.example.javakotlindemo.app;

public class AppMain {
    public static void main(String[] args) {
        System.out.println(sum(1, 2));
        System.out.println(sum(sum(3,4), 2));
    }

    private static int sum(int a, int b) {
        return a + b;
    }

}

File: src/main/java/com/example/javakotlindemo/app/AppMain.kt
package com.example.javakotlindemo.app

object AppMain {
    @JvmStatic
    fun main(args: Array<String>) {
        println(sum(1, 2))
        println(sum(sum(3, 4), 2))
        val a: Int
        a = 0
        println(a)
    }

    private fun sum(a: Int, b: Int): Int {
        return a + b
    }
}